src/test/java/e2e/apitest/UserCanAuthorizedApiTest.java

```diff
@@ -0,0 +1,53 @@
+
+package e2e.apitest;
+
+import api.AuthorizationApi;
+import api.UserApi;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import schemas.UserDto;
+
+public class UserCanAuthorizedApiTest {
+
+    UserApi userApi;
+
+    AuthorizationApi authorizationApi = new AuthorizationApi();
+
+    @Test
+    public void userCanAuthorizedApiTest() {
+        // generate new user
+        UserDto newUserData = authorizationApi.generateNewRandomUser();
+
+        //register new user
+        JsonPath createdUser = authorizationApi.registerNewUser(newUserData).jsonPath();
+        String userId = createdUser.getString("userID");
+
+        // generate token
+        String token = authorizationApi.generateToken(newUserData);
+        userApi = new UserApi(token);
+        Assert.assertTrue(authorizationApi.isAuthorized(newUserData));
+
+        //check user registered
+        userApi.getUser(userId);
+
+        //get data of created user
+        JsonPath expectedCreatedUser = userApi.getUser(userId).jsonPath();
+        Assert.assertEquals(
```

Comment:
need to change this assert , because if registerNewUser and getUser methods , give you not correct response, how you understand, what are methods give you not correct response?

`JsonPath expectedCreatedUser = userApi.getUser(userId).jsonPath();` -it will actualResult  , need to rename variable 
String expectedResult  = newUserData.getUsername()

Assert.assertEquals(
 actualResult,
                expectedResult,
                actualResult + "is not equal to " + expectedResult
        );